src/_guides/language/language-tour.md

```diff
@@ -4541,7 +4541,9 @@ concurrency is error prone and can lead to complicated code.
 
 Instead of threads, all Dart code runs inside of *isolates*. 
 Each Dart isolate has a single thread of execution and
-shares no mutable objects with other isolates.  
+shares no mutable objects with other isolates.
+Spinning up multiple isolates creates multiple threads of execution,
+enabling multi-threading without the typical issues that surround it.
```

Comment:
**suggestion:** 

```suggestion
Each Dart isolate uses one execution thread and
shares no mutable objects with other isolates.
Spinning up multiple isolates creates multiple threads of execution.
This enables multi-threading without the typical issues that surround it.
```

**question:** What issues?

Reply:
The primary issue is race conditions, e.g.: https://en.wikipedia.org/wiki/Race_condition#In_software

Reply:
> Each Dart isolate uses one execution thread and

Going to keep "a single thread of execution" as is to match the language of the original issue.